app/models/concerns/foreman_puppet/extensions/host.rb

```diff
@@ -55,6 +55,15 @@ def search_by_config_group(_key, operator, value)
       end
 
       module PrependedMethods
+        def environment_id=(id)
+          self.environment = ForemanPuppet::Environment.find(id)
+          save
```

Comment:
Is this good or does it break something to have a ènvironment_id` method? 
can the `save` be improved?